internal/nearcache/record_store.go

```diff
@@ -228,7 +352,9 @@ func (rs *RecordStore) getRecordStorageMemoryCost(rec *Record) int64 {
 }
```

Comment:
can you remove the todo in  getRecordStorageMemoryCost if it is not necessary ?

Reply:
https://github.com/hazelcast/hazelcast-go-client/pull/807/commits/22d0f4598f4b66079464377bdbe5e4aee0db7a09